src/main/CMakeLists.txt

```diff
@@ -515,8 +515,5 @@ if (MSVC)
    if (NOT ${CMAKE_VERSION} VERSION_LESS "3.8.0")
       set_target_properties(${EXECUTABLE_NAME} PROPERTIES VS_DEBUGGER_WORKING_DIRECTORY "${VS_DEBUGGER_WORKING_DIRECTORY}")
    endif ()
-   if (NOT ${CMAKE_VERSION} VERSION_LESS "3.13.0")
-      set_target_properties(${EXECUTABLE_NAME} PROPERTIES VS_DEBUGGER_COMMAND_ARGUMENTS "--debug")
-   endif ()
```

Comment:
Why this change?

Reply:
Sidharth explained to me earlier that Visual Studio adds the `--debug` flag when running the app. This prevented the app from running, because the flag was not recognized. Therefore you need to tell Visual Studio every time that it should not add that flag. Now, Sidharth apparently found the root cause of that flag. 

However, the app should now run properly with this flag since PR #8294. 
I don't know if this change should be there anyway (and if so, it should probably be at least in a different commit).

Reply:
Confirmed that this change is not required because PR #8294 adds the debug flag

Reply:
Great! In that case I propose to remove this change and the rebase this PR.